src/python-common/ceph/deployment/service_spec.py

```diff
@@ -1613,6 +1625,36 @@ def validate(self) -> None:
         ):
             raise SpecValidationError("Log file directory backups can't be negative")
 
+        if (self.max_hosts_per_namespace and self.max_hosts_per_namespace < 0):
+            raise SpecValidationError("Max hosts per namespace can't be negative")
+
+        if (self.max_namespaces_with_netmask and self.max_namespaces_with_netmask < 0):
+            raise SpecValidationError("Max namespaces with netmask can't be negative")
+
+        if type(self.max_subsystems) != int:
```

Comment:
This appears to be causing `make check` failures:
```
lint: commands[0] /home/jenkins-build/build/workspace/ceph-pull-requests/src/python-common> flake8 ceph
ceph/deployment/service_spec.py:1642:12: E721 do not compare types, for exact checks use `is` / `is not`, for instance checks use `isinstance()`
ceph/deployment/service_spec.py:1648:12: E721 do not compare types, for exact checks use `is` / `is not`, for instance checks use `isinstance()`
ceph/deployment/service_spec.py:1654:12: E721 do not compare types, for exact checks use `is` / `is not`, for instance checks use `isinstance()`
ceph/deployment/service_spec.py:1660:12: E721 do not compare types, for exact checks use `is` / `is not`, for instance checks use `isinstance()`
lint: exit 1 (1.52 seconds) /home/jenkins-build/build/workspace/ceph-pull-requests/src/python-common> flake8 ceph pid=45070
lint: FAIL ✖ in 26.1 seconds
```

Reply:
@idryomov I didn't get these when I committed. Anyway, I'm about to make another change there so I'll change this too.

Reply:
Possibly PR #60587 - as a newer flake8 was needed to support newer python versions such as 3.12.

Reply:
let me know if you need help fixing these

Reply:
Already fixed in #60818 ;)